app/pages/index.js

```diff
@@ -13,10 +14,20 @@ export default withData(props => (
         body {
           margin: 2em;
         }
+        section {
+          height: calc(50vh - 4em);
```

Comment:
Not sure why there is a fixed height, and with a calc? You can do something like `padding-top: 3em;` to create responsive spacing between sections.